src/NodaTime/Duration.cs

```diff
@@ -892,15 +894,8 @@ public static Duration FromDays(double days)
         /// </summary>
         /// <param name="hours">The number of hours.</param>
         /// <returns>A <see cref="Duration"/> representing the given number of hours.</returns>
-        public static Duration FromHours(int hours)
-        {
-            Preconditions.CheckArgumentRange(
-                nameof(hours),
-                hours,
-                (long)MinDays * HoursPerDay,
-                (MaxDays + 1L) * HoursPerDay - 1);
-            return OneHour * hours;
-        }
+        public static Duration FromHours(int hours) =>
+            FromUnits(hours, nameof(hours), (long)MinDays * HoursPerDay, (MaxDays + 1L) * HoursPerDay - 1, HoursPerDay, NanosecondsPerHour);
```

Comment:
etc parentheses with `(a)*b-1` probably being clearer as either `((a)*b)-1` or `(a)*(b-1)` depending on which precedence is correct (the first?), here and below.